web/package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-prettier": "^3.1.2",
     "eslint-plugin-react": "^7.19.0",
-    "prettier": "^1.19.1",
+    "prettier": "1.19.1",
```

Comment:
I'm not sure this is adding anything. Let's remove it until it's a necessary update.

Reply:
@asasmith it was there already, but I removed it!